extensions/binding/org.eclipse.smarthome.binding.astro/src/main/java/org/eclipse/smarthome/binding/astro/internal/discovery/AstroDiscoveryService.java

```diff
@@ -61,15 +64,23 @@ public AstroDiscoveryService() {
     }
 
     @Override
+    @Activate
```

Comment:
You do not need activate and deactivate annotations, it is only required for modified.